docs/UserGuide.md

```diff
@@ -102,11 +102,38 @@ Precise expected outputs on success:
 
 ![edit format](images/addresult.png)
 
-Precise expected outputs on failure:
+* Precise expected outputs on failure:
 * If a required parameter is missing (e.g., name, email), an error message should specify which parameter is missing.
 * If a parameter is provided in an invalid format (e.g., an invalid email address), an error message should indicate the invalid format.
 * If a parameter is specified multiple times (e.g., --name John --name Doe), an error should indicate that the parameter can only be specified once.
 
+### Search by appointment day: `cal`
+
+Lists out all persons who have an appointment on the input date
```

Comment:
Full stop for consistency, minor issue! :D

Reply:
Fixed!